chatbot/app.py

```
@@ -2,89 +2,113 @@
 
     except Exception as e:
-        return jsonify({"error": str(e)})
+        return jsonify({"error": str(e)}), 500
 
-if __name__ == '__main__':
-    app.run(host="0.0.0.0", port=5000,debug=True)
+################################
+# RUN
+################################
+if __name__ == "__main__":
+    app.run(host="0.0.0.0", port=5000, debug=True)
```

Ruff (0.14.14)

[error] Possible binding to all interfaces

(S104)